palm/pilrc/Portfile

```
@@ -1,39 +1,34 @@
-PortSystem 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name			pilrc
-version			3.1
-revision		1
-categories		palm devel
-license			GPL-2+
-maintainers		nomaintainer
-description		Palm OS resource compiler
-long_description	This is a tool that turns a textual description of \
-					Palm OS forms, bitmaps, fonts, and other resources \
-					into a binary form that can be included in an \
-					application or library. The name is derived from \
-					\"PILot Resource Compiler\". This tool is used with \
-					many popular Palm OS development tools, including \
-					prc-tools and CodeWarrior for Palm OS.
-homepage		http://pilrc.sourceforge.net/
-master_sites	sourceforge
-checksums		md5 bbfea963c4e34252f634c6a39a2578b8
-platforms		darwin
+PortSystem      1.0
 
-configure.cmd	unix/configure
-configure.args	--disable-pilrcui
+name            pilrc
+version         3.2
+revision        0
+categories      palm devel
+license         GPL-2+
+maintainers     {@nkorth nkorth.com:nkorth} openmaintainer
+description     Palm OS resource compiler
+long_description    This is a tool that turns a textual description of \
+                    Palm OS forms, bitmaps, fonts, and other resources \
+                    into a binary form that can be included in an \
+                    application or library. The name is derived from \
+                    \"PILot Resource Compiler\". This tool is used with \
+                    many popular Palm OS development tools, including \
+                    prc-tools and CodeWarrior for Palm OS.
+homepage        http://pilrc.sourceforge.net/
+master_sites    sourceforge
+checksums       rmd160  e9dfb96082c7406e269ea99aec4ce2a6029e8afd \
+                sha256  f3d6ea3c77f5d2a00707f4372a212377ab7bd77b3d68c3db7e28a553b235903f \
+                size    259898
+patchfiles      fix-resource-list.diff
 
-variant gtk	{
-	depends_lib		lib:libgtk.1:gtk1
-
-	configure.args-delete	--disable-pilrcui
-	configure.args-append	--enable-pilrcui \
-							--enable-gtktest \
-							--with-gtk-prefix=${prefix}
-}
+configure.cmd   unix/configure
+configure.args  --disable-pilrcui
 
 post-destroot {
-	set docPath ${destroot}${prefix}/share/doc
-	xinstall -d -m 0755 ${docPath}
-	system "cp -R ${worksrcpath}/doc ${docPath}/${name}"
-	system "cp -R ${worksrcpath}/example ${docPath}/${name}/"
+    set docPath ${destroot}${prefix}/share/doc
+    xinstall -d -m 0755 ${docPath}
+    system "cp -R ${worksrcpath}/doc ${docPath}/${name}"
```

Lets replace the `system "cp -R"` call with the MacPorts-native `copy` call:

```suggestion
copy ${worksrcpath}/doc ${docPath}/${name}
```